.github/workflows/publish.yml

```
@@ -0,0 +1,32 @@
+name: Publish Packages
+
+on:
+  push:
+    branches:
+      - master
```

```suggestion
  workflow_dispatch:
  push:
    branches:
      - master
```
This makes it so you can manually release too